source/renderer/app/api/ada/sendAdaBugReport.js

```diff
@@ -18,21 +19,23 @@ export const sendAdaBugReport = (
   { requestFormData, application }: SendAdaBugReportRequestParams
 ): Promise<{}> => {
   const { email, subject, problem, compressedLog } = requestFormData;
-  const reportUrl = url.parse(environment.REPORT_URL);
+  const { platform, version, build, API_VERSION, REPORT_URL } = environment;
+  const buildNumber = uniq([API_VERSION, build]).join('.');
+  const reportUrl = url.parse(REPORT_URL);
 
-  let platform;
-  switch (environment.platform) {
+  let os;
+  switch (platform) {
```

Comment:
I see we have this switch in 3 different locations. Wouldn't it be better to have it in a separated file?